universal-login-example/test/services/BackupService.js

```diff
@@ -6,11 +6,14 @@ describe('BackupService', async () => {
   let identityService;
 
   before(async () => {
-    identityService = [];
-    identityService.identity = [];
-    identityService.identity.name = 'kyle';
+    identityService = {
+      identity: {
+        name: 'kyle'
```

Comment:
Is our identity service named "Kyle"?

Reply:
This is example data for testing. 
The name picked in a way to immortalize one of the great contributors of ours :)